### PR TITLE
FIX circle

### PR DESCRIPTION
**Pull request by jasmainak**
This should hopefully fix circle ... fingers crossed

### CODE REVIEW

**Comment by jasmainak** [comment]
@larsoner and @choldgraf any clues about how to make sphinx-gallery more verbose other than reverting to an older version? Some of the scripts in autoreject are longer and sphinx-gallery suppresses all the outputs from the scripts leading to time outs in Circle

**Comment by choldgraf** [comment]
hmmm, why would that lead to time-outs in circle? That's weird!

**Comment by larsoner** [comment]
Most CIs have a "no output in X minutes" decision to kill the build, I think CircleCI is 10 minutes.

Can you open an issue on the sphinx-gallery page? We should discuss solutions over there, I have a couple of ideas.

**Comment by jasmainak** [comment]
yes, exactly. I think the default used to be 5 or 10 minutes but it's configurable.

okay, I'll open an issue on sphinx-gallery

File: circle.yml [hunk 1]
```diff
@@ -6,19 +6,21 @@ dependencies:
   override:
     - pip install --upgrade pip
     - pip install sphinx matplotlib coverage Pillow
-    - pip install sphinx-gallery sphinx_bootstrap_theme
+    - git clone https://github.com/sphinx-gallery/sphinx-gallery.git
+    - cd sphinx-gallery && git fetch --tags && git fetch --tags && git checkout v0.1.11
```
Comment (agramfort) [inline circle.yml]: why twice git fetch --tags ?

Comment (jasmainak) [reply circle.yml]: oops, good point. Fixed directly in master

**Pull request merged**